test/integration/theme/index.spec.ts

```diff
@@ -22,6 +22,8 @@ test('can change between light and dark themes', async ({ page }) => {
 
   await expect(canvasBodyLocator).toHaveCSS('background-color', 'rgb(255, 255, 255)');
 
+  await editorModel.themeEditor.getByRole('button', { name: 'Add theme' }).click();
```

Comment:
We could also call this file `editor.spec.ts` or something if it helps tell it apart from `prod.spec.ts`